PP_APP/server/frontend/templates/ranking.html

```diff
@@ -1,13 +1,86 @@
 {% extends "base.html" %}
 {% load static %}
 {% block content %}
-
 <section id="about" class="about-section bg-light">
-    <div class="container">
-
-        <!-- PAGE CONTENT GOES IN HERE -->
-
-    </div>
+   <div class="container">
+      <h1 class="h3 mb-2 text-gray-800">Quick Stats</h1>
```

Comment:
The charts are empty when I looked at them; is this expected behavior when the database is auto-generated?

Reply:
you need to run:
- python manage.py migrate authtoken
- python manage.py migrate
to generate data for the DB. 
Sorry I made some assumption here. And i will update this in the PR comment.

Reply:
I re-ran it; it still populates the table but not the charts. I looked at the Console on Google Chrome to check if something strange happened and this is what I got: 

![image](https://user-images.githubusercontent.com/34168715/81510360-ad843200-92d6-11ea-8664-6dad2fea8ded.png)

Reply:
Hmm, it works on my end out of the box. But I recommend deleting the db.sqlite3 and then run those two migrate commands. It should work hopefully.

Reply:
It still doesn't work for me; interestingly, charts work on master.